wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -495,6 +562,363 @@ Disable TLS (Not Recommended)
+
+```
+curl -X POST https://localhost:8090/api/v1/transactions/fees \
+  -H "Accept: application/json; charset=utf-8" \
+  -H "Content-Type: application/json; charset=utf-8" \
+  --cacert ./scripts/tls-files/ca.crt \
+  -d '{
+  "destinations": [{
+      "amount": 14,
+      "address": "A7k5bz1QR2...Tx561NNmfF"
+  }],
+  "source": {
+    "accountIndex": 0,
+    "walletId": "Ae2tdPwUPE...8V3AVTnqGZ"
+  }
+}'
+```
+
+The API resolves with an estimated amount in _ADA_. This estimation highly depends on the
+current state of the ledger and diverges with time.
+
+```json
+{
+    "status": "success",
+    "data": {
+        "estimatedAmount": 194
+    },
+    "meta": { ... }
+}
+```
+
+
+Managing Accounts
+-----------------
+
+A wallet isn't limited to one account. It can actually be useful to have more than one account
+in order to separate business activities. With the API, you can retrieve a specific account,
+create new ones, list all existing accounts of a wallet or edit a few things on an existing
+account. By default, your wallet comes with a provided account; let's see how to create a fresh
+new account on a wallet using [`POST /api/v1/wallets/{{walletId}}/accounts`](#tag/Accounts%2Fpaths%2F~1api~1v1~1wallets~1{walletId}~1accounts%2Fpost):
+
+```
+curl -X POST \
+  https://localhost:8090/api/v1/Ae2tdPwUPE...8V3AVTnqGZ/accounts \
+  -H 'Content-Type: application/json;charset=utf-8' \
+  -H 'Accept: application/json;charset=utf-8' \
+  --cacert ./scripts/tls-files/ca.crt \
+  -d '{
+  "name": "MyOtherAccount",
+  "spendingPassword": "5416b2988745725998907addf4613c9b0764f04959030e1b81c603b920a115d0"
+}'
+```
+
+Note that the `spendingPassword` here should match the one provided earlier in [Creating a
+New Wallet](#section/Getting-Started/Creating-a-New-Wallet).
+
+
+```json
+{
+    "status": "success",
+    "data": {
+        "index": 2902829384,
+        "addresses": [
+            "DdzFFzCqrh...Sj5a2rkx8y"
+        ],
+        "amount": 0,
+        "name": "MyOtherAccount",
+        "walletId": "Ae2tdPwUPE...8V3AVTnqGZ",
+    },
+    "meta": { ... }
+}
+```
+
+You can always retrieve this account description later if needed via [`GET /api/v1/wallets/{{walletId}}/accounts/{{accountId}}`](#tag/Accounts%2Fpaths%2F~1api~1v1~1wallets~1{walletId}~1accounts~1{accountId}%2Fget).
+
+For example:
+
+```
+curl -X GET \
+  https://127.0.0.1:8090/api/v1/wallets/Ae2tdPwUPE...8V3AVTnqGZ/accounts/2902829384 \
+  -H 'Accept: application/json;charset=utf-8' \
+  --cacert ./scripts/tls-files/ca.crt
+```
+
+For a broader view, the full list of accounts of a given wallet can be retrieved using [`GET /api/v1/wallets/{{walletId}}/accounts`](#tag/Accounts%2Fpaths%2F~1api~1v1~1wallets~1{walletId}~1accounts%2Fget)
+```
+curl -X GET \
+  https://127.0.0.1:8090/api/v1/wallets/Ae2tdPwUPE...8V3AVTnqGZ/accounts \
+  -H 'Accept: application/json;charset=utf-8' \
+  --cacert ./scripts/tls-files/ca.crt
+```
+
+```json
+{
+    "status": "success",
```

Comment:
Idea for the future: we should come up with a way to embed automatically-generated JSON in this documentation, so that if we change the API the changes are reflected automatically in this documentation.

Reply:
Are you thinking of using some `Arbitrary` instance to generate examples here (The way we do it with Swagger shemas)?
If so, this makes a lot of sense indeed.

Reply:
I was thinking something even simpler: a bunch of "exemplar" dummy types we "splice back" into the documentation. Pseudo-Haskell:

```haskell
data FooBar = Foo {
  bar :: Bar
  baz :: Baz
}

instance JSON a => Documentable a where
    spliceIntoDocumentation :: Proxy a -> Q Expr

ourSwaggerDoc = [text|

This is the example returned by a call to /v1/api/foo-bar:

$(spliceIntoDoc @FooBar Proxy)

```

I mean, you get the idea. There are lots of different ways to accomplish the above, but the idea would be to rely on _types_ not on free form strings to keep our documentation updated if we change either the JSON serialisation format or the type itself :)

Reply:
This is a fantastic idea! 

Before we go *too* much further with homebrewing a document system, though, we might want to consider the `shakespeare` libraries -- they are, at heart, a system for interpolating Haskell values into text like this. The HTML/CSS/JS sublanguages aren't necessary.

Reply:
Yup, 100% agree, and we can indeed tackle these bits outside this PR 😉

Reply:
See my last comment. Examples are now generated from `Arbitrary` instances at least for keeping all the examples in sync with the API responses. We can iterate from there to also generate the example cURL commands and / or the associated endpoint :+1:.
For now however, I think it was necessary as the doc was already getting quite a few inconsistencies and it's not even merged :| ...

Reply:
> Examples are now generated from Arbitrary instances at least for keeping all the examples in sync with the API responses.

Ah, but be careful there -- I cannot check all the details (in the middle of something else) but the reason why we added the `Example` typeclass was because not all the `Arbitrary` instances we had in the wallet yielded useful/valid data, so we had to use the `Example` typeclass to overcome it.

In this [linked issue](https://iohk.myjetbrains.com/youtrack/issue/CSL-2034) I give a (hopefully) excellent rationale for that 😉